tests/test_toasted_marshmallow.py

```diff
@@ -7,23 +7,51 @@
 @pytest.fixture()
 def schema():
     class TestSchema(Schema):
-        key = fields.String()
-        value = fields.Integer()
+        key = fields.String(default='world')
+        value = fields.Integer(missing=42)
     return TestSchema()
 
 
-def test_marshmallow_integration(schema):
-    schema.jit = toastedmarshmallow.Jit
+@pytest.mark.parametrize('jit', [toastedmarshmallow.Jit,
```

Comment:
General question: is there a no-op jit (or `None`?) that you can test with to make sure the behavior agrees with marshmallow everywhere?